src/index.js

```diff
@@ -109,14 +116,17 @@ class CreditCardInput extends Component<Props, State> {
   cardExpiryField: any;
   cardNumberField: any;
   cvcField: any;
+  zipField: any;
 
   static defaultProps = {
     cardCVCInputRenderer: inputRenderer,
     cardExpiryInputRenderer: inputRenderer,
     cardNumberInputRenderer: inputRenderer,
+    cardZipInputRenderer: inputRenderer,
     cardExpiryInputProps: {},
     cardNumberInputProps: {},
     cardCVCInputProps: {},
+    cardZipInputProps: {},
     cardImageClassName: '',
```

Comment:
Can we perhaps have an extra prop called `enableZipInput`? 😄To cater for billing countries that don't require a zip code.

Reply:
Would you prefer the default prop for `enableZipInput` to be false or true?

Reply:
Hmm, lets leave it as false? ☺️

Reply:
Hmm, lets leave it as false? ☺️